### PR TITLE
Use map instead of list for CommandFunction

### PR DESCRIPTION
**Pull request by louxiu**

### CODE REVIEW

File: src/main/java/net/glowstone/command/minecraft/FunctionCommand.java [hunk 1]
```diff
@@ -37,13 +37,7 @@ public boolean execute(CommandSender sender, String label, String[] args) {
...
         String functionName = args[0];
-        CommandFunction function = null;
-        for (CommandFunction f : world.getFunctions()) {
-            if (f.getFullName().equals(functionName)) {
-                function = f;
-                break;
-            }
-        }
+        CommandFunction function = world.getFunctions().get(functionName);
         if (function == null) {
```
Comment (aramperes) [inline src/main/java/net/glowstone/command/minecraft/FunctionCommand.java]: Use `getFunctions().containsKey()` check here, and `get` the function after that check.

**Review by aramperes — CHANGES_REQUESTED** [review]

**Comment by louxiu** [comment]
@momothereal done

**Review by aramperes — APPROVED** [review]
Looks good! Please avoid re-writing commits/history though, splitting in multiple commits is easier for reviewing changes - we squash everything when merging anyways :)

**Comment by louxiu** [comment]
@momothereal ok

**Pull request merged**